pkg/neonvm/controllers/vmmigration_controller.go

```diff
@@ -254,21 +242,31 @@ func (r *VirtualMachineMigrationReconciler) Reconcile(ctx context.Context, req c
 		return r.updateMigrationStatus(ctx, migration)
 	}
 
-	if migration.Status.Phase == "" {
-		// need change VM status asap to prevent autoscaler change CPU/RAM in VM
-		// but only if VM running
-		if vm.Status.Phase == vmv1.VmRunning {
-			vm.Status.Phase = vmv1.VmPreMigrating
-			if err := r.Status().Update(ctx, vm); err != nil {
-				log.Error(err, "Failed to update VM status to PreMigrating", "Status", vm.Status.Phase)
-				return ctrl.Result{}, err
-			}
-			// Migration just created, change Phase to "Pending"
-			migration.Status.Phase = vmv1.VmmPending
-			return r.updateMigrationStatus(ctx, migration)
+	if vm.Status.MigrationName == "" {
+		if vm.Status.Phase != vmv1.VmRunning {
+			// VM must be scaling, so we will try again in a second
```

Comment:
question:

> VM must be scaling

Why?

Reply:
That's an evolution of the `// some other VM status (Scaling may be), requeue after second`

But I agree, "may be" should not have turned into `must be`.